File: mobile-client/ios/App/Catalog/CatalogCommerceRouteMap.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
// Canonical catalog commerce route map.
public struct CatalogCommerceRouteMap: Sendable {
    public init() {}

    public func primaryFlows() -> [CatalogCommerceRoute] {
        [.vendor, .product]
    }
}

File: mobile-client/android/app/src/main/java/app/mobiling/client/catalog/CatalogCommerceRouteMap.kt
package app.mobiling.client.catalog

/**
 * Marketing America Corp. Oleksandr Tishchenko
 *
 * Canonical catalog commerce route map.
 */
class CatalogCommerceRouteMap {
    fun primaryFlows(): List<CatalogCommerceRoute> = listOf(
        CatalogCommerceRoute.VENDOR,
        CatalogCommerceRoute.PRODUCT,
    )
}
